xwiki-platform-core/xwiki-platform-security/xwiki-platform-security-authentication/xwiki-platform-security-authentication-ui/src/main/resources/XWiki/Authentication/Administration.xml

```diff
@@ -36,7 +36,73 @@
   <minorEdit>false</minorEdit>
   <syntaxId>xwiki/2.1</syntaxId>
   <hidden>true</hidden>
-  <content>{{velocity}}
+  <content>== {{translation key="security.authservice.admin.title"/}}
```

Comment:
missing closing `==` ? + I think the use of the translation macro in heading is causing a webstandard issue

Reply:
iirc because the computation of the heading id is done before the computation of the translation macro, leading to an empty id, which is raised as an error by webstandard.

Reply:
see https://github.com/xwiki/xwiki-platform/commit/6ed8244be36cee8caad055fe123b796d7337491a#diff-14c2728e791b12a9ea983da33040f3b43edd6bf16f4b5943d69b5901e5fa0638R106 for instance